nixos/modules/config/gtk/gtk-icon-cache.nix

```diff
@@ -52,10 +52,8 @@ with lib;
 
     environment.extraSetup = ''
       # For each icon theme directory ...
-
-      find $out/share/icons -mindepth 1 -maxdepth 1 -print0 | while read -d $'\0' themedir
+      find $out/share/icons -exec test -d {} ';' -mindepth 1 -maxdepth 1 -print0 | while read -d $'\0' themedir
```

Comment:
```suggestion
      find $out/share/icons -type d -mindepth 1 -maxdepth 1 -print0 | while read -d $'\0' themedir
```

Reply:
`-type d` wouldn't work there as symlinks to directories would be ignored.`-xtype d` would work but I went with `-exec test -d {} ';'` as the former isn't POSIX compliant. I'm fine either so feel free to request changes for the one you prefer.

Reply:
> would work but I went with `-exec test -d {} ';'` as the former isn't POSIX compliant.

That doesn't really matter for nixpkgs because we always use gnu tools.